*: Rephrase readme community and quickstart section (#23942)
pingcap/tidb: README.md

For more details and latest updates, see [TiDB docs](https://docs.pingcap.com/tidb/stable) and [release notes](https://docs.pingcap.com/tidb/dev/release-notes).

## Community

You can join these groups and chats to discuss and ask TiDB related questions:

- [Contributors Mailing list](https://lists.tidb.io/g/contributors)

> - [Slack Channel](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)

Reviewer: Can we change this to [Slack Channel (in Chinese and English)](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)?

Author: Why do you want to add this message? Generally I hope our channel in English with optional Chinese only channel. This is the default understanding and we don't have to mention it intentionally.

Reviewer: Yes. You are right.

> - [Chinese User Group](https://asktug.com)

Reviewer 2: It's the Chinese forum, actually not Chinese user group, maybe you mean.

Author: I'd like to learn a bit about the background. The [original text](https://github.com/pingcap/tidb/pull/23942/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L79) name AskTUG as "User Group(Chinese)" and TUG means TiDB User Group. Shall we refer AskTUG as Chinese forum consistently?

Reviewer 2: I think we should

Author: ```suggestion
- [Chinese Forum](https://asktug.com)
```

Author: @winkyao addressed.

> In addition to that, you may enjoy following:

Reviewer 3: ```suggestion
In addition to groups and chats above, you may enjoy following:
```
`that` is not clear about what is referenced

Author: I'd prefer just "In addition, you may enjoy following:"

Author: ```suggestion
In addition, you may enjoy following:
```

- [@PingCAP](https://twitter.com/PingCAP) on Twitter
- Question tagged [#tidb on StackOverflow](https://stackoverflow.com/questions/tagged/tidb)
- The PingCAP Team [English Blog](https://en.pingcap.com/blog) and [Chinese Blog](https://pingcap.com/blog-cn/)

> - [TiDB Monthly](https://pingcap.com/weekly/)

Reviewer 3: ```suggestion
- [TiDB Monthly Update](https://pingcap.com/weekly/)
```
according to things in https://pingcap.com/weekly

Author: Monthly already means Monthly update(a.k.a "月报"). Weekly / Monthly is an idiom.

> For support, please contact [PingCAP](http://bit.ly/contact_us_via_github).

Reviewer 3: ```suggestion
For enterprise support, please contact [PingCAP](http://bit.ly/contact_us_via_github).
```
The link is for enterprise support.

Reviewer 3: And why use short link?

Author: I don't think it must be "enterprise" support. Generally speaking I'd like to drop this message because TiDB is a community effort, but let's remain as is for now.

Author: > And why use short link?

I have no idea and suggest remain as is.

Reviewer 3: > I don't think it must be "enterprise" support.
Me neither. What I mean is that, when someone wants general non-enterprise support, this link gives the wrong answer :/

## Quick start

### To start using TiDB

See [Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/).

### To start developing TiDB

> If you want to build TiDB right away there are two options:

Reviewer 3: ```suggestion
If you want to build TiDB right away, there are two options:
```

- [Contributors Mailing list](https://lists.tidb.io/g/contributors)
- [Slack Channel](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)

> - [Chinese Forum](https://asktug.com)

Reviewer: Can we change this to [Forum (Currently in Chinese)](https://asktug.com)?

Author: I don't think it is a good idea until we have a real English forum or at least a clear plan.

It is in fact a Chinese forum. Why do you want to make the change?

Reviewer: You are right.
